crates/trippy-tui/locales/app.yml

```diff
@@ -830,11 +933,13 @@ column_nat:
   pt: "Nat"
   zh: "网络地址转换"
   sv: "Nat"
+  ru: "Nat"
 column_fail:
   en: "Fail"
   fr: "Échec"
   tr: "Başarısız"
   it: "Falliti"
   pt: "Falha"
   zh: "失败"
-  sv: "Misslyckades"
\ No newline at end of file
+  sv: "Misslyckades"
```

Comment:
another language

Reply:
It is just the change in EOL as it is no longer the final line in the file, no worries!

Reply:
got it